src/pages/MapViewer/components/Toolbox/components/ToolboxItems/ToolboxItem.tsx

```diff
@@ -130,7 +130,7 @@ const ToolboxItem = ({ item }: ToolboxItemProps) => {
         />
 
         <ClipboardButton text={url} />
-        <Preview item={item} />
+        {item.assets.thumbnail && <Preview item={item} />}
```

Comment:
So I've seen an issue with this, which is also true with your previous PR:

it's not always true that `assets.thumbnail` is where the preview will be.
For example, in this [Landsat STAC](https://planetarycomputer.microsoft.com/api/stac/v1/collections/landsat-c2-l2/items/LC09_L2SP_067044_20241117_02_T2) it is called 'rendered_preview'.

We are going to need a function that goes through a few variations to find the presence of a thumbnail given the STAC metadata.

One option can be that there's an asset called `thumbnail`, which seems safe to assume. Another could be that inside the `assets.roles` there is `overview` or `thumbnail`